Content.Server/Chemistry/EntitySystems/SolutionContainerSystem.cs

```diff
@@ -272,7 +272,7 @@ public bool TryAddSolution(EntityUid targetUid, Solution? targetSolution, Soluti
     /// <param name="name">name for the solution</param>
     /// <param name="solutionsMgr">solution components used in resolves</param>
     /// <returns>solution</returns>
-    public Solution EnsureSolution(EntityUid uid, string name,
+    public Solution EnsureSolution(EntityUid uid, string name, out bool alreadyExisted,
```

Comment:
might want to upstream the EnsureSolution improvement, small things like that are something they actually merge relatively quickly

Reply:
https://github.com/space-wizards/space-station-14/pull/12902